### DISCUSSION TITLE
Erreur Precondition synapps

### DISCUSSION THREAD

**Discussion by Guillaume31a** [witsa/synapps#638] [Q&A] [answered]
Bonjour, je rencontre un problème lors de la publication de mon synapps, en effet, j'ai le message d'erreur suivant : 
![image](https://github.com/witsa/synapps/assets/131860668/e8286719-5bba-47cf-b912-22d3ef450517)
Ma version de synapps est la V1.1.6 et celle de WIT est la dernière version soit la 14.4.3.
Lorsque je clique sur synapps dans l'explorateur, je n'ai rien : 
![image](https://github.com/witsa/synapps/assets/131860668/05a4e1e6-b893-4304-a166-bc10ff4c0e19)

![image](https://github.com/witsa/synapps/assets/131860668/01ee6ed3-c71c-4f14-b18b-c37befcdf167)

Merci d'avance pour votre aide.

**Comment by Raphwit** [comment, marked as answer]
Bonjour, pour ce type d'erreur, je vous invite à vérifier l'intégrité de la synapp (vérifier que les scènes possèdent bien des clés différentes notamment), c'est une erreur qui sera plus lisible à l'avenir !

**Reply by Guillaume31a** [reply]
Bonjour, merci pour votre réponse, j'ai vérifié les clés des scènes synapps comme ceci : 
![image](https://github.com/witsa/synapps/assets/131860668/945e3994-fe18-4b42-a611-3c6ac137e14b)
Puis : 
![image](https://github.com/witsa/synapps/assets/131860668/514523a3-b174-4956-81a0-ecd2aa428e73)
Sur les acteurs : 
![image](https://github.com/witsa/synapps/assets/131860668/15cb172b-45b2-4c8e-9b0f-eef60727636c)
![image](https://github.com/witsa/synapps/assets/131860668/ca28e7e0-aa9c-4637-b335-c652e3d4181d)
J'ai des clés différentes pour toutes les scènes sans caractères spéciaux.

Peut être qu'il y a une modif à faire ou un paramètre à configurer dans la prog WIT ou synapps.

Merci d'avance

**Reply by Guillaume31a** [reply]
Petite précision, j'ai essayé de publier le projet synapps sur un autre projet WIT, et ça ne fonctionne pas. Le problème vient donc de synapps.

**Reply by Guillaume31a** [reply]
C'est bon j'ai trouvé, erreur toute bête, il y a 2 fois CTA 1 ;
![image](https://github.com/witsa/synapps/assets/131860668/f44c299e-182d-483a-bf3b-156589aa5e70)
il faudrait que je change mes lunettes...
